docs/topics/mta-review-reports.adoc

```diff
@@ -152,6 +155,56 @@ image::3-5-expanded-issue.png[Expanded rule in the issues report]
 
 Issues are sorted into four categories by default. Information on these categories is available at ask Category.
 
+
+=== Insights
+
+Insights provide helpful information about the technologies used in the application and their usage in the code.
+For example, a rule searching for a deprecated API usage in the code that does not impact the current migration but can be tracked and fixed when needed in the future.
+
+Unlike issues, insights do not need to be fixed for a successful migration. They are generated by any tagging or non-tagging rule and do not have an effort and category assigned. They might have a message and tag.
+
+[NOTE]
+=====
+Insights are generated automatically, if applicable or present. However, you must run the application analysis by using the CLI because MTA 7.1 does not support their generation through the UI.
+=====
+
+You can view Insights in the Static report under the Insights tab and in the analysis output under the Insights section.

```

Comment:
`Static` shouldn't be capitalized right?

Reply:
done.